config/pipelines/detect_burnin.pipe

```diff
@@ -75,12 +75,23 @@ process detect_burnin__dilater
   filter:vxl_morphology:kernel_radius   = 2.0
 
 # =============================================================================
-process detect_burnin__out
+process detect_burnin__convert
```

Comment:
So... this should probably be named  `detect_burnin__mask_out`, and the other `detect_burnin__inpainted_out`, or something, so that it's more obvious how to get both outputs. (The GUI needs both!)

Reply:
Updated to `detect_burnin__mask`. If you feel strongly that the `_out` postfix is useful, I'll add it in a couple of other places as well.

Reply:
I would actually leave it `out`. The reason is that `out` is... the output of the "superprocess". Similarly, the new `.pipe` should have an `in` and an `out`, even if that requires creating a "dummy" process. (Which it looks like it will...)

Reply:
This may be one of the key difference between "clusters" and including pipe files.  In a cluster we define the inputs and outputs of the cluster as if it was just a process, but these ports are really just and alias to ports on internal nodes, so there is no "dummy" process created.  The cluster defines what inputs and outputs are accessible and hide the rest.  When we are just including a pipe file that whole pipeline is exposed and anyone can tap into any ports they want.  It might be that for our current applications the outputs are the mask and inpainted results, but these have no special status.  Someone can just as easily tap into any port of the included file. 

I think it's better for the processes to be named in a way that documents what they do rather that how we anticipate this pipe file might be used by someone who includes it.

Reply:
The problem with that approach is that it requires the user to have knowledge of the internal implementation of the `.pipe`. That's fine if you *want* to inspect the guts, but if you just want to use the thing, it makes it more complicated and less obvious what you're trying to do at the point of use.

That said, I think what would be useful in the long term is some sort of hybrid, where we can define a "cluster" that allows poking at the guts (and that follows the config search paths rather than having its own, independent system), but also allows some sort of scoping of its internals with the ability to specify "outward-facing" ports.

Reply:
I see the appeal of simply naming it `out` but I also think that it's a bit weird to give most things a descriptive name and then have the only thing the user touches have no semantic meaning. Furthermore, unless we move toward better documenting things, you'd still need to inspect an embedded pipeline to determine the types of the ports you're getting, so it seems reasonable to figure out the name as well. `<super process>__<semantic>_out` seems like a reasonable convention if we want to clue users in to things which may be useful. But I think `<super process>__<semantic>` is nearly as good and might be cleaner if there are multiple optional outputs. 

I'll look into creating a dummy process for the inpainter input.